elastic/kibana
# [Entity Analytics] Require full risk engine privileges to enable, disable and init the risk engine.

## Summary

Previously, a user only required SO read/write to enable or disable the risk engine once it had been initialised. This PR makes it so that a user requires all risk engine privileges to do any of these actions. 

**Test steps**

A user is required to have cluster manage_index_templates and manage_transform permissions as well as read and write access to `'risk-score.risk-score-*'` in order to access these APIs.

1. Create a user with a privilege missing
2. Attempt one of the following URL calls:
    - `POST kbn:/internal/risk_score/engine/init`
    - `POST kbn:/internal/risk_score/engine/enable`
    - `POST kbn:/internal/risk_score/engine/disable`
3. All should return a 403 with an informative error message about which permission is missing.
